package.json

```diff
@@ -56,10 +56,15 @@
     "debug-break": "node --nolazy --inspect-brk scripts/kibana --dev",
     "precommit": "node scripts/precommit_hook",
     "karma": "karma start",
-    "lint": "echo 'use `node scripts/eslint` and/or `node scripts/tslint`' && false",
-    "lintroller": "echo 'use `node scripts/eslint --fix` and/or `node scripts/tslint --fix`' && false",
-    "makelogs": "echo 'use `node scripts/makelogs`' && false",
-    "mocha": "echo 'use `node scripts/mocha`' && false",
+    "lint": "node scripts/eslint && node scripts/tslint && node scripts/sasslint",
+    "lint:es" : "node scripts/eslint",
+    "lint:ts" : "node scripts/tslint",
+    "lint:sass" : "node scripts/sasslint",
+    "lint:fix": "node scripts/eslint --fix && node scripts/tslint --fix",
+    "lint:es:fix" : "node scripts/eslint --fix",
```

Comment:
I think we can remove the fix - you can call `lint:es --fix`

Reply:
@tylersmalley While thats true, its not documented. Is there another place where we could make this option obvious to devs?

Reply:
It's in the help text:

```
Fixing problems:
  --fix                          Automatically fix problems
  --fix-dry-run                  Automatically fix problems without saving the changes to the file system
```

Additionally, if we don't call out the option in the lint results, we could add it there. Adding mappings to each command line argument can't scale.